### ISSUE TITLE
No such file or directory: 'preload_HT_3mm'

### ISSUE THREAD

**Issue by farahanams** [banilo/nips2015#1]
I have this error. Is this should be my own work?

**Comment by banilo** [comment]
Yes, exactly. Simply use NiftiMasker and a brain template to get a 2D array
that will replace the 'preload_HT_3mm' dump. I hope this helps.

Farahana <notifications@github.com> schrieb am Mo., 3. Apr. 2017 um
07:49 Uhr:
